### ISSUE TITLE
Suggestion: warn about redundant parens in LambdaCase patterns

### ISSUE THREAD

**Issue by sinelaw** [ndmitchell/hlint#185]
```haskell
{-# LANGUAGE LambdaCase #-}
module Foo where
foo :: Maybe a -> Int
foo = \case
    (Nothing) -> 0
    _ -> 1
```

hlint should warn about the parenthesis around "Nothing" being redundant.

**Comment by ndmitchell** [comment]
Thanks for the report. I've merged this into an existing issue - https://github.com/ndmitchell/hlint/issues/113#issuecomment-157993132.
